### ISSUE TITLE
Add a screenshot to readme.md

### ISSUE THREAD

**Issue by TheInitializer** [decent-chat/decent#147]
Would be nice for people to see what decent looks like.

Maybe wait until the web client's in a more finished state?

**Comment by PullJosh** [comment]
Honestly, I think that the default client (which needs a name, by the way), should eventually be moved into its own separate repo. It makes sense to keep them together for now (because pretty much all API changes are directly related to a change in the client-side code), but eventually it should be a [submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules) (another repo).

Anyway, it makes sense to include a screenshot *after* that switch is made.

**Comment by TheInitializer** [comment]
After #172 it looks like we've decided to keep everything in this one repo.

**Comment by heyitsmeuralex** [comment]
Assigning @towerofnix because they have a Mac <sup>something something nonfree</sup>, which always gives super nice single-window high DPI screenshots 🎉 

Also I'm planning to create a few themes soon (see: userstyles) so it'd be nice to include screenshots of them in the docs somewhere :)

**Comment by heyitsmeuralex** [comment]
Note that this should be regularly updated in the future!

**Comment by towerofnix** [comment]
@heyitsmeuralex I have a Mac[book pro with retina display] but refuse to use it. 🎉

**Comment by heyitsmeuralex** [comment]
@towerofnix Can you survive the nonfree for a few seconds to take a screenshot or no? 📦

**Comment by towerofnix** [comment]
🚱 :us:

<img width="964" alt="coolpic" src="https://user-images.githubusercontent.com/9948030/34962799-a4e538ac-fa1b-11e7-94e5-2b1fe26ea824.png">

**Comment by heyitsmeuralex** [comment]
Hey, uh...

shadows?

**Comment by towerofnix** [comment]
Yeah, but the shadows on the screenshot were like 80px big. 10px would be nicer.

**Comment by heyitsmeuralex** [comment]
Could you perhaps take it without the other tab? 😛

**Comment by towerofnix** [comment]
It would be nice if meta.decent.chat was fixed, so I could have a better example screenshot 😃

**Comment by towerofnix** [comment]
<img width="914" alt="coolpic2" src="https://user-images.githubusercontent.com/9948030/34963464-1387bd04-fa1f-11e7-99f2-c84c833bc84c.png">

**Comment by ktrvs** [comment]
I could take this, how would you like it played out? For example, would you like a "screenshots" title or just have the screenshot put under another section?

**Comment by heyitsmeuralex** [comment]
@ktrvs we're waiting on a fixed client before we take a screenshot. It can go near the top of the README.

**Comment by ktrvs** [comment]
Ah, Okay. 👍
